pkgs/tools/archivers/arc_unpacker/default.nix

```diff
@@ -0,0 +1,33 @@
+{ stdenv, fetchFromGitHub, cmake, writeText, boost, libpng, libjpeg, zlib
+, openssl, libwebp, catch }:
+
+stdenv.mkDerivation rec {
+  pname = "arc_unpacker";
+  version = "b9843a13e2b67a618020fc12918aa8d7697ddfd5";
+
+  src = fetchFromGitHub {
+    owner = "vn-tools";
+    repo = pname;
+    rev = version;
+    sha256 = "0wpl30569cip3im40p3n22s11x0172a3axnzwmax62aqlf8kdy14";
+  };
+
+  nativeBuildInputs = [ cmake catch ];
+  buildInputs = [ boost libpng libjpeg zlib openssl libwebp ];
+
+  patches = [ ./fix-asset-path.patch ];
```

Comment:
Should add an explanation as to why this patch is being used

Reply:
Removed the patch and kept the directory structure by putting everything in `libexec` instead.